code/components/extra-natives-five/src/EntityExtraNatives.cpp

```diff
@@ -4,6 +4,50 @@
 #include <scrBind.h>
 
 #include <Streaming.h>
+#include <Hooking.h>
+
+static int PedIdOffset;
+static uintptr_t CReplayPedExtensionPtr;
+
+#pragma pack(push, 1)
+class OverlayEntry
+{
+public:
+	uint32_t collection_hash; // 0x0000
+	uint32_t overlay_hash; // 0x0004
+	char pad_0008[8]; // 0x0008
+	float unk_float; // 0x0010
+}; // Size: 0x0014
+
+class PedEntry
+{
+public:
+	char pad_0000[184]; // 0x0000
+	OverlayEntry entries[87]; // 0x00B8
+	uint32_t overlay_count; // 0x0784
+	char pad_0788[80]; // 0x0788
+}; // Size: 0x07D8
+
+class CReplayPedExtension
+{
+public:
+	PedEntry entries[1]; // Indexed by a value from CPed-class (0x308)
+};
+
+class ptrCReplayPedExtension
+{
+public:
+	class CReplayPedExtension* ptr; // 0x0000
+}; // Size: 0x0008
+#pragma pack(pop)
+
+static struct DecorationResult

```

Comment:
`static` is a bit weird for something that's not declaring anything.

Reply:
Monkey see, monkey do. I used existing FiveM code as a reference. But can change up.

https://github.com/citizenfx/fivem/commit/bc2f29bbc57da75a423c77fb760380f230b07b24